BrainPortal/app/views/signups/new.html.erb

```diff
@@ -24,180 +24,167 @@
+        %>
+
+
+        <%= f.label(:email, "* Institutional Email address:") %>
+        <%= f.text_field :email, :size => 60 %>
+
+        <p></p>
+        <p>
+          <small>Please supply the address of <strong class="warning"> your research institution.</strong>
+            Requests with non institutional address or email
+            will be <strong>ignored</strong>
           </small>
-      </td>
-    </tr>
+        </p>
+
+        <%= f.label(:street1, "Institution Street Address (line 1):") %>
+        <%= f.text_field :street1, :size => 60 %>
 
-    <% if current_user.present? && current_user.has_role?(:admin_user) %>
-      <tr>
-        <td colspan="2">&nbsp;</td>
-      </tr>
+        <%= f.label(:street2, "Institution Street Address (line 2):") %>
+        <%= f.text_field :street2, :size => 60 %>
 
-      <tr>
-        <th> <%= f.label(:admin_comment, "Private, admin-only comments:") %> </th>
-        <td> <%= f.text_area :admin_comment, :cols=>80, :rows => 5 %> </td>
-      </tr>
-    <% end %>
+        <%= f.label(:city, "* City of Institution:") %>
+        <%= f.text_field :city, :size => 30 %>
 
-    <tr>
-      <td colspan="2">&nbsp;</td>
-    </tr>
 
-    <tr>
-      <td colspan="2" class="centered">
-        <%= f.submit(@signup.new_record? ? "Request Account" : "Update Request") %>
-      </td>
-    </tr>
+        <%= f.label(:province, "* Province/State:") %>
+        <%= f.text_field :province, :size => 30 %>
+
+        <%= f.label(:country, "* Country:") %>
+        <%= f.text_field :country, :size => 30 %>
+
+        <%= f.label(:postal_code, "Postal/ZIP Code:") %>
+        <%= f.text_field :postal_code, :size => 8 %>
+
+        <%= f.label(:time_zone, "Time Zone:") %>
+
+        <%= f.time_zone_select :time_zone,
+                               ActiveSupport::TimeZone.all.select {|t| t.name =~ /canada/i},
+                               {:default => ActiveSupport::TimeZone['Eastern Time (US & Canada)'], :include_blank => true}
+        %>
 
-  </table>
 
-  <% end %>
+        <%= f.label(:comment, "Comments or special requests:") %>
+
+        <%= f.text_area :comment, :cols => 80, :rows => 5 %>
+        <br>
+
+        <small>
+          Please tell us the name of the laboratory you work for, the name of its Principal Investigator (if not you),
+          and if possible anyone<br>
+          else you know in your lab who are already CBRAIN users. We'll use this information to create or add you to a
+          <em>Site</em> within CBRAIN.
+        </small>
+
+
+      <% if current_user.present? && current_user.has_role?(:admin_user) %>
+
+          <%= f.label(:admin_comment, "Private, admin-only comments:") %>
+          <%= f.text_area :admin_comment, :cols => 80, :rows => 5 %>
+
+      <% end %>
+
+      </div>
+
+      <br>
+    </fieldset>
+    <br>
+    <div class="centered">
+      <%= f.submit(@signup.new_record? ? "Request Account" : "Update Request") %>
+    </div>
+
+    </div>
```

Comment:
Your divs are not matched! Check this carefully!

Reply:
This is why the privacy note ends up all formatted wrong with this code in place

Reply:
did not notice any difference with privacy note format. Perhaps you'd like get it into grey box?